spec/Spec/Polynomials.cry

```diff
@@ -51,3 +52,18 @@ evaluate_polynomial_in_evaluation_form polynomial z =
             width' = toInteger width
             r = (pow_mod z' width') - 1
             evaluation = (results!0) * r * inverse_width
+
+/**
+ * The BLS basic data blob
+ * @see https://github.com/ethereum/consensus-specs/blob/dev/specs/deneb/polynomial-commitments.md#custom-types
+ * Also @see https://github.com/ethereum/consensus-specs/blob/dev/specs/deneb/polynomial-commitments.md#blob
+ */
+type BlsBlob = [BYTES_PER_FIELD_ELEMENT * FIELD_ELEMENTS_PER_BLOB][8]
```

Comment:
nit: The spec calls this just plain `Blob` but I'm guessing this is a much more useful name for distinguishing blob types.

Reply:
Yeah, I waffled on this. It also calls `Polynomial` instead of `BlsPolynomial` like I do.

I think the `BlsFieldElement` prefix is needed because there are other fields `Fq`, `Fq2`, etc., but these other ones are the only "Blobs" and "Polynomials."

I'm starting to lean towards just make it reflect the spec ... What do you think / prefer, should I drop the `Bls` prefix?

Reply:
Taking the plunge.